src/agents/read_agent.py

```
@@ -1,51 +1,143 @@
-"""Read agent for data retrieval operations."""
+"""Read agent for data retrieval operations with LangChain tools."""
 
 import logging
 from typing import Dict, Any, List
-from langchain_core.messages import HumanMessage
+from langchain_core.prompts import ChatPromptTemplate
```

Ruff (0.12.2)

`langchain_core.prompts.ChatPromptTemplate` imported but unused

Remove unused import: `langchain_core.prompts.ChatPromptTemplate`

(F401)